Website/src/app/components/iconSelection.tsx

```diff
@@ -1,13 +1,11 @@
 import Select, { Options, SingleValue } from "react-select";
 import { Option } from "@/utils/types";
 import { useMemo } from "react";
+import { POIIconCommonName, POIIconImg } from "@/utils/common";
+import { POITypeIcon, POITypeIconValues } from "@/utils/api";
+import Image from "next/image"
 
-export const icons = [
-	{ path: "/poiTypeIcons/generic_rail_bound_vehicle.svg", name: "Schienenfahrzeug" },
-	{ path: "/poiTypeIcons/level_crossing.svg", name: "Bahnübergang" },
-	{ path: "/poiTypeIcons/lesser_level_crossing.svg", name: "Unbeschilderter Bahnübergang" },
-	{ path: "/poiTypeIcons/parking.svg", name: "Haltepunkt" }
-];
+const POI_ICONS: POITypeIcon[] = Object.values(POITypeIconValues);
```

Comment:
This would have been much more messy, and much less type-safe with a typescript enum.